varya/assets/css/variables.css

```diff
@@ -163,13 +163,19 @@
 	--branding--logo--max-width-mobile: 96px;
 	--branding--logo--max-height-mobile: 96px;
 	--primary-nav--font-family: var(--global--font-secondary);
-	--primary-nav--font-size: 16px;
+	--primary-nav--font-family-mobile: var(--global--font-primary);
+	--primary-nav--font-size: var(--global--font-size-sm);
+	--primary-nav--font-size-mobile: var(--global--font-size-xxl);
+	--primary-nav--font-size-sub-menu-mobile: var(--global--font-size-lg);
+	--primary-nav--font-style: normal;
+	--primary-nav--font-style-sub-menu-mobile: italic;
```

Comment:
Some of these new variables feel bit granular, but I included them because I figure we'll want to override them pretty easily for child themes.

Reply:
Yeah, at first I thought this might’ve been too complex, but its actually kind of necessary to easily make a ‘Vanilla’ version. 👍